GitHub.Collectors/Collector/PointCollector.cs

```diff
@@ -20,7 +20,7 @@ public class PointCollector
     {
         private readonly CollectorBase<GitHubCollectionNode> collector;
         private readonly ICache<PointCollectorTableEntity> cache;
-
+        private static string storageAccountNameEnvironmentVariable;
```

Comment:
>private static string storageAccountNameEnvironmentVariable; [](http://example.com/codeflow?start=8&length=60)

why static?

Reply:
because the method OffloadToPointCollector is static and cannot accept non-static variables